packages/core/src/lib/disk.ts

```diff
@@ -1,24 +1,42 @@
+import { debuglog } from 'node:util'
```

Comment:
This is super nice. It prints debug traces scoped by a string. I saw Flydrive used it